database.py

```diff
@@ -0,0 +1,23 @@
+import sqlite3
+
+import config
+
+# connection is a static variable that is shared across all instances of DB
+# so we reuse the same connection across all users of this class
+class Database():
+    connection = None
+    def _init__(self):
+        if Database.connection is None:
```

Comment:
also, i think Database would be better suited as an abstraction to the underlying db, instead of manual sqlite statements all over the codebase that become harder to check correctness of (checking all sql is injection safe in one file is alot easier than jumping around)

Reply:
Making it __init__ would mean we try to create the tables every time we instantiate a Database. Indent the table-creation code by one level, and that should be fixed.

Reply:
we can have the *check* if its None in the `__init__`
it wont try and make the db every time then

Reply:
fixed the typo and now create the tables once per new connection